internal/coremsgs/en_config_descriptions.go

```diff
@@ -20,301 +20,265 @@ import (
+	ConfigBlockchainFabricFabconnectPrefixShort  = ffc("config.blockchain.fabric.fabconnect.prefixShort", "The prefix that will be used for Fabconnect specific query parameters when FireFly makes requests to Fabconnect", stringType)
+	ConfigBlockchainFabricFabconnectSigner       = ffc("config.blockchain.fabric.fabconnect.signer", "The Fabric signing key to use when submitting transactions to Fabconnect", stringType)
+	ConfigBlockchainFabricFabconnectTopic        = ffc("config.blockchain.fabric.fabconnect.topic", "The websocket listen topic that the node should register on, which is important if there are multiple nodes using a single Fabconnect", stringType)
+	ConfigBlockchainFabricFabconnectURL          = ffc("config.blockchain.fabric.fabconnect.url", "The URL of the Fabconnect instance", "URL "+stringType)
+
+	ConfigBlockchainFabricFabconnectProxyURL = ffc("config.blockchain.fabric.fabconnect.proxy.url", "The URL for the Fabconnect proxy", "URL "+stringType)
+
+	ConfigBroadcastBatchAgentTimeout = ffc("config.broadcast.batch.agentTimeout", "How long to keep around a batching agent for a sending identity before disposal", stringType)
+	ConfigBroadcastBatchPayloadLimit = ffc("config.broadcast.batch.payloadLimit", "The maximum payload size of a batch for broadcast messages", byteSizeType)
+	ConfigBroadcastBatchSize         = ffc("config.broadcast.batch.size", "The maximum number of messages that can be packed into a batch", intType)
+	ConfigBroadcastBatchTimeout      = ffc("config.broadcast.batch.timeout", "The timeout to wait for a batch to fill, before sending", timeDurationType)
+
+	ConfigCorsCredentials = ffc("config.cors.credentials", "CORS setting to control whether a browser allows credentials to be sent to this API", booleanType)
+	ConfigCorsDebug       = ffc("config.cors.debug", "Whether debug is enabled for the CORS implementation", booleanType)
+	ConfigCorsEnabled     = ffc("config.cors.enabled", "Whether cors is enabled", booleanType)
+	ConfigCorsHeaders     = ffc("config.cors.headers", "CORS setting to control the allowed headers", stringType)
+	ConfigCorsMaxAge      = ffc("config.cors.maxAge", "The maximum age a browser should rely on CORS checks", timeDurationType)
+	ConfigCorsMethods     = ffc("config.cors.methods", " CORS setting to control the allowed methods", stringType)
+	ConfigCorsOrigins     = ffc("config.cors.origins", "CORS setting to control the allowed origins", stringType)
+
+	ConfigDatabaseMaxChartRows = ffc("config.database.maxChartRows", "The maximum rows to fetch for each histogram bucket", intType)
+	ConfigDatabaseType         = ffc("config.database.type", "The type of the database interface plugin to use", intType)
+
+	ConfigDatabasePostgresMaxConnIdleTime = ffc("config.database.postgres.maxConnIdleTime", "The maximum amount of time a database connection can be idle", timeDurationType)
```

Comment:
Might be worth a `global.` for future database plugins

Reply:
I actually thought of and tried this. Without introducing some sort of wildcard key searching it doesn't work with the way globals currently work, because `postgres` is in the middle of the key.

So `config.database.myKey` would match `config.global.myKey`
But there's no way to match `config.database.postgres.myKey` to `config.database.*.myKey` or anything like that right now. I'm not opposed to adding this if we really want it to eliminate a few keys here.